test/program_test.cpp

```diff
@@ -27,7 +27,11 @@ migraphx::program create_program()
 TEST_CASE(program_equality)
 {
     migraphx::program x = create_program();
-    migraphx::program y = create_program();
+    migraphx::program y;
+
+    y = x;
```

Comment:
Why is this changed?

Reply:
I had thought it will cover the lines:
    else if(!impl->modules.empty())
    {
        impl->modules.clear();
    }
But looks like it does not go here.